tests/docker/test_cli.py

```diff
@@ -0,0 +1,63 @@
+def test_generate_dataset_in_container(capsys, run_in_container):
+    run_in_container(
+        [
+            "generate_dataset",
+            "--help",
+        ]
+    )
+
+    captured = capsys.readouterr()
+    assert captured.out
+    assert not captured.err
```

Comment:
Are you planning further validation here and in the other tests below?

Reply:
I wasn't, no.  Do you think we need anything more than an exception isn't thrown?